backend/nodes/image_filter_nodes.py

```diff
@@ -325,9 +327,9 @@ def __init__(self):
             is guaranteed to be normalized."""
         self.inputs = [
             ImageInput("Normal Map 1"),
-            SliderInput("Strength 1", 0, 100, 100),
+            SliderInput("Strength 1", minimum=0, maximum=100, default=100),
```

Comment:
Same here, please make minimum=0 the default, so we don't have to repeat it everywhere.